site/docs/components/splitter/index.mdx

```diff
@@ -4,7 +4,7 @@ data:
   description: "`Splitter` divides window content into separate regions called split panels that can be dragged and resized, allowing users to customize the layout of their workspace."
   sourceCodeUrl: "https://github.com/jpmorganchase/salt-ds/tree/main/packages/lab/src/splitter"
```

Comment:
sourceCodeUrl contains a lab reference to source code

Reply:
Done!